server/services/sentence-plan/stepsService.ts

```diff
@@ -9,6 +9,11 @@ export default class StepService {
     return restClient.post<Step[]>({ path: `/goals/${parentGoalId}/steps`, data: steps })
   }
 
+  async saveAllSteps(steps: NewStep[], parentGoalId: string) {
+    const restClient = await this.sentencePlanApiClient.restClient('Saving multiple steps')
```

Comment:
Could we have a different message here from saveSteps please?

Reply:
Fixed in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/123/commits/242550e04b40b03017145a91dd94ba08a6d1e628